core-bundle/src/Migration/AbstractRecordedMigration.php

```diff
@@ -47,6 +52,21 @@ protected function createResult(bool $successful, string $message = null): Migra
         return parent::createResult($successful, $message);
     }
 
+    private function createMigrationTable(): void
+    {
+        $entityManager = clone $this->entityManager();
```

Comment:
If we don’t remove any listeners I guess we don’t have to clone the entity manager?

```suggestion
        $entityManager = $this->entityManager();
```